src/main/java/org/isf/medicalstockward/service/MedicalStockWardIoOperations.java

```diff
@@ -120,6 +123,29 @@ public int getCurrentQuantityInWard(
 
 		return (int) (mainQuantity != null ? mainQuantity : 0.0);	
 	}
+	
+	/**
+	 * Gets the current quantity for the specified {@link Ward} and {@link Lot}.
+	 * @param ward - if {@code null} the quantity is counted for the whole hospital
+	 * @param lot - the {@link code}
+	 * @return the total quantity.
+	 * @throws OHServiceException if an error occurs retrieving the quantity.
+	 */
+	public int getCurrentQuantityInWard(
+			Ward ward, 
+			Lot lot) throws OHServiceException 
+	{
+		Double quantity;
+		if (ward!=null) 
```

Comment:
Formatting doesn't match standards; braces are not on their own line.

Reply:
Fixed